mne/event.py

```diff
@@ -109,25 +109,32 @@ def define_target_events(events, reference_id, target_id, sfreq, tmin, tmax,
     lag : ndarray
         time lag between reference and target in milliseconds.
     """
+    
+    if isinstance(reference_ids, int):
+        reference_ids = [reference_ids]
 
-    if new_id is None:
-        new_id = reference_id
+    if isinstance(target_ids, int):
+        target_ids = [target_ids]
 
     tsample = 1e3 / sfreq
     imin = int(tmin * sfreq)
     imax = int(tmax * sfreq)
 
     new_events = []
     lag = []
+
     for event in events.copy().astype(int):
-        if event[2] == reference_id:
+        if event[2] in reference_ids:
             lower = event[0] + imin
             upper = event[0] + imax
+            matching_events = np.any([events[:, 2] == target for target in targets_ids])
```

Comment:
no but it's maybe more readable with set. Something like:

len(set(events[:, 2]).intersection(targets_ids)) > 0